bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -127,6 +151,10 @@
 		<label>Rain Intensity</label>
 		<description>Current precipitation intensity.</description>
 		<category>Rain</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Speed</tag>
```

Comment:
For a similar channel measuring rain intensity in bundles/org.openhab.binding.metofficedatahub/src/main/resources/OH-INF/thing/channel-types.xml it was:
```suggestion
			<tag>Rain</tag>
```

Not sure which is correct, but it should probably be consistent?